app/frontend/src/cljs/teet/file/file_view.cljs

```diff
@@ -609,31 +612,15 @@
                                          :entity-id (:db/id @selected-file)
                                          :show-comment-form? false}]])]])))
 
-(defn file-part-heading
-  [{heading :heading
-    number :number} opts]
-  [:div {:style {:margin-bottom "1.5rem"
-                 :display :flex
-                 :justify-content :space-between}}
-   [:div {:class (<class common-styles/flex-row-end)
-          :style {:margin-right "0.5rem"}}
-    [typography/Heading2 {:style {:margin-right "0.5rem"}}
-     heading]
-    (when number
-      [typography/GrayText
-       {:style {:white-space :nowrap}}
-       (goog.string/format "#%02d" number)])]
-   [:div
-    (when-let [action-comp (:action opts)]
-      action-comp)]])
-
 (defn no-files
   []
-  [typography/GrayText {:class [(<class typography/gray-text-style)
+  [typography/GrayTextDiv {:class [(<class typography/gray-text-style)
                                 (<class common-styles/flex-row)
                                 (<class common-styles/margin-bottom 1)]}
-   [ti/file {:style {:margin-right "0.5rem"}}]
-   [:span (tr [:file :no-files])]])
+   [:div {:class [(<class common-styles/flex-row)
+                  (<class common-styles/margin-bottom 1.5)]}
+    [ti/file {:style {:margin-right "0.5rem"}}]
+   [:span {:style {:margin-top "2px"}} (tr [:file :no-files])]]])
```

Comment:
maybe use justify-content: center, or align-items center instead of hard-coding some margin-top value in pixels.